Source/missiles.cpp

```diff
@@ -689,7 +689,7 @@ bool GuardianTryFireAt(Missile &missile, Point target)
 	return true;
 }
 
-bool GrowWall(int id, Point position, Point target, MissileID type, int spellLevel, int damage)
+bool GrowWall(int id, Point position, Point target, MissileID type, int spellLevel, int damage, bool skip = false)

```

Comment:
```suggestion
bool GrowWall(int id, Point position, Point target, MissileID type, int spellLevel, int damage, bool addMissile = true)
```

`skip` is a little bit generic.

Alternative Names:
- `addMissile` (and invert if)
- `onlyCheckCanGrow`

Reply:
How about `skipTile`?

Reply:
`skipTile` = skip grow wall check or skip creating the missile? 😉 

That's why I would prefer something related to the missile / firewall
- `placeWallSegment`
- `placeWall`
- `skipWallSegment`
- `skipWall`

?

Reply:
Good point. `skipWall` is good.